.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__/
```

Comment:
I believe this should be a part of [a global gitignore file](https://stackoverflow.com/a/7335487). I would suggest you to remove the `.gitignore`.